src/vue/backend/src/components/BulkAction.vue

```diff
@@ -2,7 +2,19 @@
     <div>
         <idx-block className="import-header__description">{{ description }}</idx-block>
         <idx-block className="import-header__actions-bar">
-            <idx-block className="import-header__select-all" @click="$emit('select-all', selected)">{{ selected ? 'Deselect All' : 'Select All' }}</idx-block>
+            <idx-block
+                className="import-header__select-all import-header__select-all--loading"
+                v-if="loading"
+            >
+                {{ selected ? 'Deselect All' : 'Select All' }}
+            </idx-block>
+            <idx-block
+                className="import-header__select-all"
+                @click="$emit('select-all', selected)"
```

Comment:
You can combine these two to avoid the v-of v-else pattern. Something like:
```
<idx-block
    :className="{
          import-header__select-all: true
          import-header__select-all--loading: loading
     }"
    @click=" loading ? false : $emit('select-all', selected)"
>
    {{ selected ? 'Deselect All' : 'Select All' }}
</idx-block>
```